tests/vmi_bridge_test.go

```diff
@@ -0,0 +1,480 @@
+/*
+ * This file is part of the kubevirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	expect "github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	appsv1 "k8s.io/api/apps/v1"
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/log"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("Bridge", func() {
+
+	bridgeIP := map[string]string{"red": "172.16.98.1", "blue": "172.16.99.1"}
+
+	flag.Parse()
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+	tests.BeforeAll(func() {
+		// create a deamon set for the network bridge device plugin
+		// taken from here: https://github.com/kubevirt/kubernetes-device-plugins/blob/master/manifests/bridge-ds.yml
+		// when running on OpenShift, we have to make sure that we have right privileges to make the pods privileged
+		runNetworkBridgeDevicePlugin := func() {
+			const repo = "quay.io/kubevirt"
+			const tag = "latest"
+			const name = "device-plugin-network-bridge"
+			const userName = "device-plugin-admin"
+
+			// Create service account for the daemon set
+			// Needed only for OpenShift, but should work on k8s as well
+			serviceAccount := k8sv1.ServiceAccount{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      userName,
+					Namespace: tests.NamespaceTestDefault,
+					Labels: map[string]string{
+						v1.AppLabel: "test",
+						"name":      name,
+					},
+				},
+			}
+			_, err = virtClient.CoreV1().ServiceAccounts(tests.NamespaceTestDefault).Create(&serviceAccount)
+			Expect(err).ToNot(HaveOccurred())
+
+			// Setting SCC for the service account is only needed for OpenShift cluster
+			// Not verifying result since this is expected to fail on anything other than OpenShift
+			tests.RunKubectlCommand("adm", "policy", "add-scc-to-user", "privileged", "-n",
```

Comment:
I am trying to build this patch, but every attempt fails with:

```
tests/vmi_bridge_test.go:77:4: undefined: tests.RunKubectlCommand
```

Is there a method missing, am I doing something wrong or is this OpenShift specific?

Reply:
@gonzolino I think it is caused by this pr https://github.com/kubevirt/kubevirt/pull/1408

Reply:
Ah yes thank for the hint! Exchanging `RunKubectlCommand` with `RunCommand("kubectl"` did the trick 👍